llvm/lib/CodeGen/ExpandFp.cpp

```diff
@@ -62,8 +62,8 @@ static cl::opt<unsigned>
 ///   br i1 %cmp6.not, label %if.end12, label %if.then8
 ///
 /// if.then8:                                         ; preds = %if.end
-///   %cond11 = select i1 %tobool.not, i64 9223372036854775807, i64 -9223372036854775808
-///   br label %cleanup
+///   %cond11 = select i1 %tobool.not, i64 9223372036854775807, i64
+///   -9223372036854775808 br label %cleanup
```

Comment:
I get this was automatically reflowed with clang-format, but some of the commented IR becomes unreadable as a result.